inc/toolbox.class.php

```diff
@@ -1105,12 +1114,15 @@ static function commonCheckForUseGLPI($isInstall = false) {
 
    /**
     * Check SELinux configuration
+    * returned values meaning:
+    *    0: OK
+    *    1: Warning
+    *    2: Error
     *
-    * @since 0.84
-    * @param $fordebug    Boolean true is displayed in system information
+    * @since version 0.84
     *
```

Comment:
Seems like you removed documentation of `$fordebug` param.

Reply:
thx